storage/innobase/include/ut0ut.h

```diff
@@ -88,15 +88,25 @@ struct ut_when_dtor {
    the YieldProcessor macro defined in WinNT.h. It is a CPU architecture-
    independent way by using YieldProcessor. */
 #  define UT_RELAX_CPU() YieldProcessor()
-# elif defined(HAVE_ATOMIC_BUILTINS)
```

Comment:
Here you have replaced HAVE_ATOMIC_BUILTINS to __powerpc__  is HAVE_ATOMIC_BUILTINS defined only on __powerpc__ or can this be executed on other platforms before the change ?